### PR TITLE
Add plugin 'Giphy'

### PR DESCRIPTION
**Pull request by DarkNightSkies**

### CODE REVIEW

**Comment by panpawn** [comment]
you didn't write this

File: chat-plugins/wisp.js [hunk 1]
```diff
@@ -854,6 +854,54 @@ exports.commands = {
...
+					}
+				} catch (e) {
+					return self.errorReply("ERROR CODE 503: Giphy is unavaliable right now. Try again later.");
+					return room.update();
+				}
+			});
+		});
+		req.end();
+	},
+	giphyhelp: function(target, room, user) {
```
Comment (panpawn) [inline chat-plugins/wisp.js]: this isn't how we do help commands

**Comment by DarkNightSkies** [comment]
I added the box around it because I didn't like the way it showed up

**Comment by panpawn** [comment]
no need to make this help command special when PS has a built in help command system imo

**Comment by DarkNightSkies** [comment]
Okay ;-;

**Comment by aakashrajput** [comment]
This plugin looks like animal.is

**Comment by DarkNightSkies** [comment]
Kyvn and I made i didn't think Animals suited it right so I changed the name

**Comment by SerperiorBae** [comment]
I'm not happy with this being used without my permission.
You literally contributed one line of code and attempted to help me with formatting.
Please ask in the future; removing you from the credits because I dont use that line of code anymore.

**Comment by panpawn** [comment]
Sounds like this is being taken care of in https://github.com/jd4564/Pokemon-Showdown/pull/82 ?_?

**Pull request closed without merging**